workflow/rocoto/gefs_tasks.py

```diff
@@ -308,22 +310,35 @@ def atmos_ensstat(self):
         return task
 
     def wavepostsbs(self):
+
         deps = []
-        dep_dict = {'type': 'metatask', 'name': f'gefs_fcst_mem#member#'}
+        dep_dict = {'type': 'task', 'name': f'gefs_fcst_mem#member#_#seg_dep#'}
         deps.append(rocoto.add_dependency(dep_dict))
         dependencies = rocoto.create_dependency(dep=deps)
 
+        fhrs = self._get_forecast_hours('gefs', self._configs['wavepostsbs'], 'wave')
+        is_replay = self._configs['wavepostsbs']['REPLAY_ICS']
+        if is_replay:
+            fhrs = [fhr for fhr in fhrs if fhr not in [0, 1, 2]]
```

Comment:
Can you place a comment on what this is doing and why?

Reply:
Done